### ISSUE TITLE
Project still active?

### ISSUE THREAD

**Issue by flynneva** [open-needs/open-needs.github.io#2]
This project seems extremely interesting and am just wondering if it is still active at all since the last commit seems to be from awhile ago?

If it is still active, what's the plan for it going forward?

**Comment by danwos** [comment]
The idea is not dead, but unfortunately, I need to focus on other Open-Source projects and I'm missing currently a project, which wants to use it and provides some use cases.

But I'm still watching it, and any PR gets merged

**Comment by flynneva** [comment]
@danwos awesome to hear! Are there any open issues / a roadmap for missing features that I could look at to see where I could best help out?

**Comment by danwos** [comment]
Not really, there are some open issues in the related repositories of the server and client. But that's far away from some sort of release/feature planning.

The best way would be to install everything and see if it already fits the needs of a project, which wants to use it.

Or you could take a look into the features of [Sphinx-Needs](https://github.com/useblocks/sphinx-needs) and start migrating the most important ones also to Open-Needs.

Or you could take another documentation generator as Sphinx, and write there a small extension, like https://github.com/open-needs/sphinx-open-needs
